bundle/Resources/config/services.yaml

```diff
@@ -1,176 +1,7 @@
-            - "@knp_menu.renderer_provider"
-            - "@ibexa.config.resolver"
-            - "@fos_http_cache.http.symfony_response_tagger"
-
-    ngsite.controller.relations.related_items:
-        class: Netgen\Bundle\SiteBundle\Controller\Relations\RelatedItems
-        public: true
-        parent: ngsite.controller.base
-        arguments:
-            - "@ngsite.relation.location_resolver"
-
-    ngsite.controller.relations.related_multimedia_items:
-        class: Netgen\Bundle\SiteBundle\Controller\Relations\RelatedMultimediaItems
-        public: true
-        parent: ngsite.controller.base
-        arguments:
-            - "@ngsite.relation.multimedia_resolver"
-
-    ngsite.controller.download:
-        class: Netgen\Bundle\SiteBundle\Controller\Download
-        public: true
-        parent: ngsite.controller.base
-        arguments:
-            - "@netgen.ibexa_site_api.site"
-            - "@ibexa.field_type.ezbinaryfile.io_service"
-            - "@ibexa.field_type.ezimage.io_service.published"
-            - "@translator"
-            - "@event_dispatcher"
-
-    ngsite.controller.user.register:
-        class: Netgen\Bundle\SiteBundle\Controller\User\Register
-        public: true
-        parent: ngsite.controller.base
-        arguments:
-            - "@ibexa.api.service.user"
-            - "@ibexa.api.service.content_type"
-            - "@event_dispatcher"
-            - "@ibexa.api.repository"
-            - "@ibexa.config.resolver"
-
-    ngsite.controller.user.activation_form:
-        class: Netgen\Bundle\SiteBundle\Controller\User\ActivationForm
-        public: true
-        parent: ngsite.controller.base
-        arguments:
-            - "@ibexa.api.service.user"
-            - "@event_dispatcher"
-            - "@ibexa.config.resolver"
-
-    ngsite.controller.user.activate:
-        class: Netgen\Bundle\SiteBundle\Controller\User\Activate
-        public: true
-        parent: ngsite.controller.base
-        arguments:
-            - "@ibexa.api.service.user"
-            - "@event_dispatcher"
-            - "@ngsite.repository.user_accountkey"
-            - "@ibexa.api.repository"
-            - "@ibexa.config.resolver"
-
-    ngsite.controller.user.forgot_password:
-        class: Netgen\Bundle\SiteBundle\Controller\User\ForgotPassword
-        public: true
-        parent: ngsite.controller.base
-        arguments:
-            - "@ibexa.api.service.user"
-            - "@event_dispatcher"
-            - "@ibexa.config.resolver"
-
-    ngsite.controller.user.reset_password:
-        class: Netgen\Bundle\SiteBundle\Controller\User\ResetPassword
-        public: true
-        parent: ngsite.controller.base
-        arguments:
-            - "@ibexa.api.service.user"
-            - "@event_dispatcher"
-            - "@ngsite.repository.user_accountkey"
-            - "@ibexa.api.repository"
-            - "@ibexa.config.resolver"
+imports:
+    - { resource: services/**/*.yaml }
```

Comment:
Why not achieve this through DI extension, something like https://github.com/netgen/better-ibexa-admin-ui/blob/master/bundle/DependencyInjection/NetgenBetterIbexaAdminUIExtension.php#L35

Reply:
That looks pretty much the same to me, or even more complicated while we still have `services.yaml`.

Reply:
With DI extension, all registration of YAML files is contained in one place. That's the only reason why I prefer it over importing here.

e.g.

```
$loader->load('services.yaml');
$loader->load('services/*.yaml', 'glob');
```

First line/file is for general purpose or one-off services that don't require a separate services file, and the second one is to actually separate chunks of related services.

Reply:
Updated in https://github.com/netgen/site-bundle/pull/39/commits/5a11bcbb1028813aeb586de889b67bd20f5462d2